web/src/features/fba/pages/FireBehaviourAdvisoryPage.tsx

```diff
@@ -152,17 +154,49 @@ const FireBehaviourAdvisoryPage: React.FunctionComponent = () => {
     document.title = ASA_DOC_TITLE
   }, [])
 
+  const formControlRef = useRef<HTMLDivElement>(null)
+  const mapRef = useRef<HTMLDivElement>(null)
+  const navRef = useRef<HTMLDivElement>(null)
+  const sidePanelRef = useRef<HTMLDivElement>(null)
+  const [formControlHeight, setFormControlHeight] = useState<number>(0)
+  const [navRefHeight, setNavRefHeight] = useState<number>(0)
+
+  useEffect(() => {
+    if (navRef.current) {
+      setNavRefHeight(navRef.current.clientHeight)
+    }
+  }, [navRef.current?.clientHeight])
+
+  useEffect(() => {
+    if (formControlRef.current) {
+      setFormControlHeight(formControlRef.current.clientHeight)
+    }
+  }, [formControlRef.current?.clientHeight])
+
+  useEffect(() => {
+    const sidePanelElement = sidePanelRef.current
+    const mapElement = mapRef.current
+    if (sidePanelElement && mapElement && formControlHeight && navRefHeight) {
+      const height = `calc(100vh - ${formControlHeight + navRefHeight}px)`
+      sidePanelElement.style.height = height
+      mapElement.style.height = height
+    }
+  })
+
   return (
     <div className={classes.root}>
-      <GeneralHeader
-        isBeta={true}
-        spacing={1}
-        title={FIRE_BEHAVIOUR_ADVISORY_NAME}
-        productName={FIRE_BEHAVIOUR_ADVISORY_NAME}
-      />
+      <Container disableGutters maxWidth={'xl'}>
+        <GeneralHeader
```

Comment:
Continuing on from my comment above about an alternate approach to setting the height of the component.

You could add a `height` prop to the `<GeneralHeader>`. The value would be `calc(100vh - ${formControlHeight + navRefHeight}px)` and this could be assigned to the top level element of the GeneralHeader component like this ```<Container sx={{height: props.height}} className=`${props.className} ${classes.wrapper}`>```